lib/resources/forms.js

```diff
@@ -45,12 +46,16 @@ const streamAttachment = async (container, attachment, response) => {
   } else {
     const dataset = await Datasets.getById(attachment.datasetId, true).then(getOrNotFound);
     const properties = await Datasets.getProperties(attachment.datasetId);
-    const lastModifiedTime = await Datasets.getLastUpdateTimestamp(dataset.id);
 
+    const actorId = !dataset.ownerOnly || auth == null || (await auth.can('entity.list', dataset))
+      ? null
+      : auth.actor.get().id;
+
+    const lastModifiedTime = await Datasets.getLastUpdateTimestamp(dataset.id);
```

Comment:
This line didn't change, I think it's just a quirk of the diff. In the next PR, `actorId` will feed into `lastModifiedTime`. So it's important to get `actorId` first.